lib/logger.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+// var exists = require('101/exists');
+// var isObject = require('101/is-object');
+var bunyan = require('bunyan');
+// var Bunyan2Loggly = require('bunyan-loggly').Bunyan2Loggly;
+
+/**
+ * Bunyan logger for ponos.
+ * @author Bryan Kendall
+ * @module ponos:logger
+ */
+module.exports = bunyan.createLogger({
```

Comment:
for now, I think leaving Ponos to do it's own logging to stdout and (optionally) loggly is good. Tasks that are defined by users cano handle any error reporting they need.